docs/policy-only.md

```diff
@@ -0,0 +1,169 @@
+
+#### Conntrack
+Note also that with re-entrance traffic a service request, after being load balanced and routed
+back to OVS bridge via ``gw0``, has exactly the same 5-tuple as when it re-enters the host network
+for forwarding.
+
+When a service request with same 5-tuples re-enters the host network, it confuses Linux conntrack. 
+The Linux considers the re-entrance IP packet from a new connection flow that uses same source port
+that has been allocated in the DNAT connection. In turn, the re-entrance packet triggers
+another SNAT connection. The overall effect is that the service's DNAT connection is not
+discovered by the service reply, and no Un-DNAT takes place. As the result, the reply is not
+recognized, and therefore dropped by the source Pod.
+   
+Antrea uses the following mechanisms to handle Pod-to-Service traffic re-entrance to the host
+network, and bypasses conntrack in host network.
+1. In OVS bridge, adds flow that marks any re-entrance traffic with a special source MAC.
+1. In OVS bridge, adds flow that causes any re-entrance traffic to bypasses conntrack in OVS zone.
+1. In the host network' ip-tables, adds a rule in ``raw`` table that if matching the special
+source MAC in 1), bypass conntrack in host zone.
+
+#### NetworkPolicy Considerations
+Note that when a traffic flow is re-entrance, the original reply packets do not make into OVS,
+it is un-DNATted in the host network before reaching OVS. Does it have any impact on NetworkPolicy
+enforcement? 
+ 
+Antrea enforces NetworkPolicy by allowing or disallowing initial connection packets (e.g. TCP
+ sync) to go through and to establish connection. Once a connection is
+established, Antrea relies on conntrack to admit or reject packets for that connection. This still 
+holds true for re-entrance traffic flows, except that conntrack takes place not within OVS conntrack
+zone, but instead is in the host network's default conntrack zone. Hence NetworkPolicy
+enforcement is not impacted. 
+
+It has some effects on statistics collection. If original reply traffic reaches OVS bridge as is
+the case of encap traffic flows, the OVS bridge knows about any reply packets dropped by OVS zone
+conntrack, and can record them accordingly. With re-entrance traffic, the reply traffic with
+original server Pod IPs does not reach OVS bridge, and any dropped traffic by host network
+conntrack is unknown to the OVS bridge.  
+
+## Additional Works
+1. Smoother transition in/out of Antrea in policy mode, Kubernetes deployment shall be easily
+scaled up and down after/before Antrea insertion to allow Pods be added added to Antrea after 
+installation, and reconnect to old CNI topology after Antrea is uninstalled.
+1. Reconciliation during mode change. Up to this point, encap is the pre-dominate mode. As more
+traffic mode is used, it is reasonable for a cluster to change traffic encapsulation on the 
+flight, Some components routeClient, ovsConfigClient(??) may not be ready for it.
```

Comment:
Remove (??) if we are going to merge it?

Reply:
done.